internal/policies/mount/adsys_mount/src/main.rs

```diff
@@ -1,22 +1,58 @@
-use clap::Parser;
 use lib::AdsysMountError;
 mod logger;
 
-/// Arguments required to run this binary
-#[derive(Debug, clap::Parser)]
-#[command(version, about, long_about = None)]
-struct Args {
-    /// Path for the file containing the mounts for the user.
-    mounts_file: String,
-}
-
 fn main() -> Result<(), AdsysMountError> {
-    let args = Args::parse();
+    if std::env::args().any(|p| &p == "--help") {
+        print_help_msg();
+        return Ok(());
+    }
+
+    let mut args = std::env::args();
+
+    // Ensures that the binary is executed with exactly 2 arguments.
+    if args.len() != 2 {
+        print_usage_msg();
+        return Err(AdsysMountError::ParseError);
+    }
+
+    // Ignores the first argument, which is the path of the executable.
+    args.next();
+
+    let mounts_file = match args.next() {
+        Some(arg) => arg,
+        None => {
```

Comment:
Now taht we check that the len is what we expected, we never can enter that brace, correct?

I wonder if we can then simplify with `if let Some(mounts_file) = args.next()…`

Reply:
Since we already ensured that we have two arguments, it's possible to just unwrap it and not worry about panicking out